pkg/cmd/codespace/create.go

```diff
@@ -48,7 +48,17 @@ func newCreateCmd(app *App) *cobra.Command {
 
 // Create creates a new Codespace
 func (a *App) Create(ctx context.Context, opts createOptions) error {
-	locationCh := getLocation(ctx, a.apiClient)
+
+	// Overrides for Codespace developers to target test environments
+	vscsLocation := os.Getenv("VSCS_LOCATION")
+	vscsTarget := os.Getenv("VSCS_TARGET")
+	vscsTargetUrl := os.Getenv("VSCS_TARGET_URL")
+
+	locationResult := locationResult{vscsLocation, nil}
+	if vscsLocation == "" {
+		// Automatic region detection
+		locationResult = <-getLocation(ctx, a.apiClient)
```

Comment:
This immediately reads from a channel that `getLocation()` creates, making the whole async aspect redundant. The original idea behind using the channel is that the rest of the Create function can proceed _while_ `getLocation()` is being looked up.